You have video processor config saved in `preprocessor.json` file which is deprecated. Video processor configs should be saved in their own `video_preprocessor.json` file. You can rename the file or load and save the processor back which renames it automatically. Loading from `preprocessor.json` will be removed in v5.0.
Loading checkpoint shards: 100%|████████████| 2/2 [00:00<00:00,  2.12it/s]
trainable params: 1,843,200 || all params: 3,756,466,176 || trainable%: 0.0491
Generating train split: 10000 examples [00:00, 45368.99 examples/s]
  0%|                                            | 0/4750 [00:00<?, ?it/s]Traceback (most recent call last):
  File "/home/jiyoon/LViton_GRPO/train.py", line 152, in <module>
    trainer.train()
  File "/home/jiyoon/.local/lib/python3.10/site-packages/transformers/trainer.py", line 2238, in train
    return inner_training_loop(
  File "/home/jiyoon/.local/lib/python3.10/site-packages/transformers/trainer.py", line 2582, in _inner_training_loop
    tr_loss_step = self.training_step(model, inputs, num_items_in_batch)
  File "/home/jiyoon/.local/lib/python3.10/site-packages/transformers/trainer.py", line 3790, in training_step
    inputs = self._prepare_inputs(inputs)
  File "/home/jiyoon/.local/lib/python3.10/site-packages/trl/extras/profiling.py", line 98, in wrapper
    return func(self, *args, **kwargs)
  File "/home/jiyoon/.local/lib/python3.10/site-packages/trl/trainer/grpo_trainer.py", line 1258, in _prepare_inputs
    generation_batch = self._generate_and_score_completions(generation_batch)
  File "/home/jiyoon/.local/lib/python3.10/site-packages/trl/trainer/grpo_trainer.py", line 1677, in _generate_and_score_completions
    rewards = (rewards_per_func * self.reward_weights.to(device).unsqueeze(0)).nansum(dim=1)
RuntimeError: The size of tensor a (4) must match the size of tensor b (3) at non-singleton dimension 1
Traceback (most recent call last):
  File "/home/jiyoon/LViton_GRPO/train.py", line 152, in <module>
    trainer.train()
  File "/home/jiyoon/.local/lib/python3.10/site-packages/transformers/trainer.py", line 2238, in train
    return inner_training_loop(
  File "/home/jiyoon/.local/lib/python3.10/site-packages/transformers/trainer.py", line 2582, in _inner_training_loop
    tr_loss_step = self.training_step(model, inputs, num_items_in_batch)
  File "/home/jiyoon/.local/lib/python3.10/site-packages/transformers/trainer.py", line 3790, in training_step
    inputs = self._prepare_inputs(inputs)
  File "/home/jiyoon/.local/lib/python3.10/site-packages/trl/extras/profiling.py", line 98, in wrapper
    return func(self, *args, **kwargs)
  File "/home/jiyoon/.local/lib/python3.10/site-packages/trl/trainer/grpo_trainer.py", line 1258, in _prepare_inputs
    generation_batch = self._generate_and_score_completions(generation_batch)
  File "/home/jiyoon/.local/lib/python3.10/site-packages/trl/trainer/grpo_trainer.py", line 1677, in _generate_and_score_completions
    rewards = (rewards_per_func * self.reward_weights.to(device).unsqueeze(0)).nansum(dim=1)
RuntimeError: The size of tensor a (4) must match the size of tensor b (3) at non-singleton dimension 1
